File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/RisorseUmane.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class RisorseUmane
    {

        /// <summary>
        /// Cod_Risum su database
        /// </summary>
        public int codice { get; set; }

        public IntervalloTemporale validita { get; set; }

        public string settore { get; set; }
        public string attivita { get; set; }

        public Contatto contatto { get; set; }

        public RapportoContabile rapportoContabile { get; set; }

        public bool flag_cancellazione { get; set; }

        public RisorseUmane(int codice)
        {
            this.codice = codice;
            this.flag_cancellazione = false;
        }

        public RisorseUmane()
        {
            this.flag_cancellazione = false;
        }

    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/RisorseUmane.ts
import { Contatto } from './Contatto';
import { IntervalloTemporale } from './IntervalloTemporale';
import { RapportoContabile } from './RapportoContabile';

export class RisorseUmane {
  codice: number;
  validita: IntervalloTemporale;
  settore: string;
  attivita: string;
  contatto: Contatto;
  rapportoContabile: RapportoContabile;
  flag_cancellazione: boolean;

  constructor() {
    this.flag_cancellazione = false;
    this.codice = 0;
    this.validita = new IntervalloTemporale();
    this.settore = '';
    this.attivita = '';
  }
}
